lib/translator/inquiry.cc

```diff
@@ -0,0 +1,252 @@
+// Copyright 2020 Google LLC
```

Comment:
Is this file mistakenly included in this PR?

Reply:
ah yep, leftover from editor